README.adoc

```diff
@@ -0,0 +1,33 @@
+= CodeReady Containers - OpenShift 4 on your Laptop
+:icons:
+:toc: macro
+:toc-title:
+:toclevels:
+
+toc::[]
+
+[[intro-to-crc]]
+== Introduction
+
+This project is focused on bringing a minimal http://github.com/openshift/origin[OpenShift 4.0 or newer] cluster to your local laptop or desktop computer. 
+
+If you are looking for a solution for running OpenShift 3.x, you will need tools such as `oc cluster up`, http://github.com/minishift/minishift[Minishift] or https://developers.redhat.com/products/cdk/overview/[CDK].
+
+Check back for updates, we plan to have something available "soon".
```

Comment:
`Check back` or `Come back` ? cc @robin-owen

Reply:
"Check back" is fine -- I'm more concerned about having quotation marks around "soon", personally.

Reply:
@robin-owen Regarding `soon` we can fix it after we have our first release.